ssc/cmod_pvwattsv7.cpp

```diff
@@ -557,6 +557,7 @@ class cm_pvwattsv7 : public compute_module
 			//   If 2 module per Y, then nmodx=nrows/2.
 			pv.nmodx = pv.nrows / pv.nmody;
 			pv.row_spacing = module.length * pv.nmody / pv.gcr;
+			ssval.init(pv.tilt, pv.gcr);
```

Comment:
If I'm understanding this function correctly, this could be sped up even further with an additional argument for fixed tilt systems- where there is no sweep of 1000 tilt values and the sky diffuse reduction is only calculated once. This would apply both to pvwattsv7 and pvsamv1.

Reply:
I think for fixed tilt, the sky diffuse derate needs to be calculated at least once. A single calculation of a sky diffuse derate value requires the sweep of 1000 tilt values.

Btw, I also added an `en_self_shading` flag to help make the initialization of sssky_diffuse_table more clear.